internal/command/routegroup/printers.go

```diff
@@ -67,14 +67,21 @@ func printRouteGroupDetails(cfg *config.RouteGroup, out io.Writer, rg *models.Ro
 
 	if len(rg.Endpoints) > 0 {
 		fmt.Fprintln(out)
-		if err := printEndpointTable(out, rg.Endpoints); err != nil {
+		if err := printEndpointTable(out, rg); err != nil {
 			return err
 		}
 	}
 
 	return nil
 }
 
+func getCluster(cluster string) string {
+	if cluster != "" {
+		return cluster
+	}
+	return "- (multi-cluster)"
```

Comment:
I would just return `"(multi-cluster)"`

Reply:
Done in https://github.com/signadot/cli/pull/207/commits/40bee4d816d1808b05cf9133904ecc616dd01d8e.